test/Common/TestHelpers.cs

```diff
@@ -20,6 +20,7 @@
 using Microsoft.WindowsAzure.Storage.Table;
 using Xunit;
 using Xunit.Abstractions;
+using static Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests.PlatformSpecificHelpers;
```

Comment:
nit: this may lead to unused using statements for FUNCTIONS_V1. Consider just prepending your static calls below with PlatformSpecificHelpers.